server/pulp/plugins/conduits/repo_sync.py

```diff
@@ -53,9 +53,13 @@ class RepoSyncConduit(RepoScratchPadMixin, ImporterScratchPadMixin, AddUnitMixin
     allowed to do whatever threading makes sense to optimize its sync process.
     Calls into this instance do not have to be coordinated for thread safety,
     the instance will take care of it itself.
+
+    :ivar importer_object_id: The ObjectId of the importer for the given repo_id
+                              and importer_id.
+    :type importer_object_id: bson.objectid.ObjectId
     """
 
-    def __init__(self, repo_id, importer_id):
+    def __init__(self, repo_id, importer_id, importer_object_id=None):
```

Comment:
I'm concerned about defaulting this to be `None`.  Since importers will depend on this to create catalog entries, I think it should be required.